src/index.html

```diff
@@ -1,13 +1,299 @@
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="menu page__menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo-link">
+            <img
+              class="top-bar__logo"
+              src="./images/logo_black.png"
+              alt="MYBIKE logo"
+            >
+          </a>
+          <a href="#" class="icon icon--close"></a>
+        </div>
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#">Home</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#about-us">About Us</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#compare-bikes">Compare Bikes</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#details">Details</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#contacts">Contacts</a>
+              </li>
+            </ul>
+          </nav>
+          <a href="tel:+1 234 5555-55-55" class="menu__phone-number">
+            +1 234 5555-55-55
+          </a>
+          <a href="tel:+1 234 5555-55-55" class="menu__book-a-ride">
+            Book a test ride
+          </a>
+        </div>
+      </div>
+    </aside>
+
+    <main class="main">
+      <div class="container">
+        <div class="main__content">
+          <section class="about-us" id="about-us">
+            <h2
+              class="about-us__title section-title section-title--align--left"
+            >
+              Move Free
+            </h2>
+            <p class="about-us__description">
+              Shift your ride, not gears. Find the fastest way to move in the
+              city as the bike adapts intuitively to power the speed you need.
+            </p>
+          </section>
+          <section class="compare-bikes" id="compare-bikes">
+            <h2
+              class="section-title section-title--margin56 compare-bikes__title"
+            >
+              Compare Bikes
+            </h2>
+            <div class="compare-bikes__bikes">
+              <article class="bike compare-bikes__bike">
+                <img
+                  src="./images/first_bike.png"
+                  alt="Sporty 4"
+                  class="bike__photo image"
+                >
+                <h3 class="bike__name">Sporty 4</h3>
+                <p class="bike__description">
+                  The iconic frame brought to a new performance height as a
+                  sporty, active ride.
+                </p>
+                <h3 class="bike__price">$ 2 590</h3>
+              </article>
+              <article class="bike compare-bikes__bike">
```

Comment:
```suggestion
              <article class="bike compare-bikes__bike">
                <img
                  src="./images/first_bike.png"
                  alt="Sporty 4"
                  class="bike__photo image"
                >
                <h3 class="bike__name">Sporty 4</h3>
                <p class="bike__description">
                  The iconic frame brought to a new performance height as a
                  sporty, active ride.
                </p>
                <h3 class="bike__price">$ 2 590</h3>
              </article>
              
              <article class="bike compare-bikes__bike">
```
add indents, now it's hard to read